.gitignore

```diff
@@ -8,3 +8,5 @@
 /build
 /dist
 /docs/_build
+.idea
```

Comment:
Maybe keep this out of the PR as it's not related

Reply:
done.